dessia_common/core.py

```diff
@@ -722,13 +723,39 @@ def save_babylonjs_to_file(self, filename: str = None, use_cdn: bool = True, deb
         """
         self.volmdlr_volume_model(**kwargs).save_babylonjs_to_file(filename=filename, use_cdn=use_cdn, debug=debug)
 
+    def to_zip_stream(self, archive: dcf.BinaryFile) -> List[dcf.BinaryFile]:
+        """ Creates a zip archive containing several files representing the export of a 3D object. """
+        step_stream = dcf.StringFile("export_step.stp")
+        self.to_step_stream(step_stream)
+        html_stream = self.to_html_stream(dcf.StringFile(filename="export_html.html"))
+        stl_stream = dcf.BinaryFile("export_stl.stl")
+        self.to_stl_stream(stl_stream)
+
+        list_stream = [step_stream, html_stream, stl_stream]
+        archive_name = 'export_zip'
+        archive.filename = archive_name
+        with ZipFile(archive, 'w') as zip_archive:
```

Comment:
This is duplicated from workflow Archive blocks and need to be mutualized. Consider adding a function to files.py module